src/Tizen.Nlp/NlpCommon/NlpCommon.cs

```diff
@@ -0,0 +1,104 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Collections;
+using Tizen;
+using System.Threading.Tasks;
+using Tizen.Applications;
+using RPCPort.message.Proxy;
+using Tizen.Applications.Messages;
+
+namespace Tizen.nlp
```

Comment:
The name of namespace should be used PascalCase.
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-namespaces